axelor-bank-payment/src/main/resources/roles/bank-payment_role.csv

```diff
@@ -1,4 +1 @@
 "name";"description"

```

Comment:
Do we not need these files? `bank-payment_role.csv`  and related change in `bank-payment-config.xml`

Reply:
This one is not needed, I merged the bank roles into accounting